src/RTCRtpSender.ts

```diff
@@ -22,6 +25,7 @@ export default class RTCRtpSender {
         this._peerConnectionId = info.peerConnectionId;
         this._id = info.id;
         this._rtpParameters = new RTCRtpSendParameters(info.rtpParameters);
+        this._dtmf = new RTCDTMFSender(info.peerConnectionId, info.id);
```

Comment:
Don't initialize this eagerly, it should be `null` if the track doesn't do DTMF, like in a sender for video: https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpSender/dtmf#value